.github/workflows/compose-fastnear-test.yml

```
@@ -0,0 +1,133 @@
+            status=$(docker inspect -f '{{.State.Health.Status}}' $service)
+            if [ "$status" != "healthy" ]; then
+              echo "Service $service is not healthy"
+              docker-compose -f ./docker-compose-fastnear.yml logs $service
+              exit 1
+            else
+              echo "Service $service is healthy"
+            fi
+          done
+
+      - name: Tear down Docker Compose services
+        run: docker-compose down
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)